.github/workflows/azure-e2e-test.yml

```diff
@@ -234,8 +242,9 @@ jobs:
     - name: Create peerpod subnet
       env:
         AZURE_LOCATION: ${{ matrix.parameters.location }}
+        AZURE_RESOURCE_GROUP: ${{ vars.AZURE_RESOURCE_GROUP }}
       run: |
-        NODE_RG="$(az aks show -g ${{ vars.AZURE_RESOURCE_GROUP }} -n "$CLUSTER_NAME" --query nodeResourceGroup -o tsv)"
+        NODE_RG="$(az aks show -g ${AZURE_RESOURCE_GROUP} -n "$CLUSTER_NAME" --query nodeResourceGroup -o tsv)"
```

Comment:
There is an unrelated previously existing inconsistency in `$CLUSTER_NAME` -> `${CLUSTER_NAME}`. Could you address that as a new commit (or do we not care?)